### ISSUE TITLE
can't install PHP 7.x

### ISSUE THREAD

**Issue by hmsun-es** [lukesampson/scoop#4189]
Recently, I can't install php 7.x after release php 8.0.

It reason of wrong php archive file name.

`line 4: php-{php-ver}-Win32-**vs16**-x86.zip`

Thank you.

```
composer install php@7.1.15
...
...
WARN  Given version (7.1.15) does not match manifest (8.0.0)
WARN  Attempting to generate manifest for 'php' (7.1.15)
Autoupdating php
Searching hash for php-7.1.15-Win32-vs16-x86.zip in https://windows.php.net/downloads/releases/sha256sum.txt
Could not find hash in https://windows.php.net/downloads/releases/sha256sum.txt
Downloading php-7.1.15-Win32-vs16-x86.zip to compute hashes!
The remote server returned an error: (404) Not Found.
URL https://windows.php.net/downloads/releases/php-7.1.15-Win32-vs16-x86.zip is not valid
Could not find hash!
Could not install php@7.1.15
```

**Comment by tomeli5n** [comment]
Hi,
You can install php7 from this bucket:
https://github.com/ScoopInstaller/PHP
